### PR TITLE
docs(templates/typescript/threejs): update for MUD 2.0 🏎

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: docs/pages/templates/typescript/threejs.mdx [hunk 1]
```diff
@@ -189,15 +191,17 @@ export const useKeyboardMovement = () => {
...
       }
-      if (e.key === "ArrowDown") {
+      if (e.key === "s") {
         moveBy(-1, 0, 0);
-      }      .
+      }
+      .
       .
       .
       .
```
Comment (holic) [inline docs/pages/templates/typescript/threejs.mdx:203]: can we use the `<CollapseCode>` component for this? that's what this is for, so we don't have to manually insert these placeholder characters

Comment (qbzzt) [reply docs/pages/templates/typescript/threejs.mdx]: Done

File: docs/pages/templates/typescript/threejs.mdx [hunk 2]
```diff
@@ -156,7 +160,7 @@ Both `useComponentValue` and `useEntityQuery` cause the React component in which
       <ambientLight />
       {/* eslint-disable-next-line react/no-unknown-property */}
       <pointLight position={[10, 10, 10]} />
-      <Plane position={[0, -5, 0]} />
+      <Plane />
```
Comment (holic) [inline docs/pages/templates/typescript/threejs.mdx:163]: mind updating this one with `CollapseCode` while you're here?

Comment (qbzzt) [reply docs/pages/templates/typescript/threejs.mdx:173]: Done.

File: docs/pages/templates/typescript/threejs.mdx [hunk 3]
```diff
@@ -147,16 +151,26 @@ const players = useEntityQuery([Has(Position)]).map((entity) => {
 Get the list of players (with the entityId and position).
 Both `useComponentValue` and `useEntityQuery` cause the React component in which they are registered (in this case, `Scene`) to be rendered again when the information changes.
 
-```ts {10-16}
-.
-.
-.
+<CollapseCode>
+
+```ts {18-24}
```
Comment (holic) [inline docs/pages/templates/typescript/threejs.mdx:156]:
```suggestion
```tsx {18-24}
```

**Pull request merged**